services/thumbnails/pkg/thumbnail/thumbnail.go

```diff
@@ -112,3 +112,21 @@ func IsMimeTypeSupported(m string) bool {
 	_, supported := SupportedMimeTypes[mimeType]
 	return supported
 }
+
+func PrepareRequest(width, height int, tType, checksum string) (Request, error) {
```

Comment:
Exported func `PrepareRequest` should have comment or be unexported

Reply:
added